mat/format.go

```diff
@@ -19,7 +19,7 @@ func Formatted(m Matrix, options ...FormatOption) fmt.Formatter {
 	for _, o := range options {
 		o(&f)
 	}
-	return f
+	return &f
```

Comment:
This change is not necessary; all the `Format` method remains on the concrete type and the value is small.